src/styles/addNewBook.css

```diff
@@ -0,0 +1,59 @@
+.new-book {
+  padding: 20px 0;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.new-book h3 {
+  color: #888;
+  padding-bottom: 0.9rem;
+}
+
+.new-book-cat {
+  height: 40px;
+  font-size: 1em;
+  padding: 0.5em;
+  border: 1px solid rgba(0, 0, 0, 0.214);
+  border-radius: 4px;
+  color: gray;
+}
+
+.new-book-title .new-book-author .new-book-cat {
+  background-color: #fff;
+  height: 30px;
+  flex-grow: 2;
+}
+
+form {
+  display: flex;
+  gap: 10px;
+}
+
+.new-book-title {
+  flex-grow: 2;
+  height: 40px;
+  font-size: 1em;
+  padding: 0.5em;
+  border: 1px solid rgba(0, 0, 0, 0.214);
+  border-radius: 4px;
+}
+
+.new-book-author {
+  height: 40px;
+  font-size: 1em;
+  padding: 0.5em;
+  border: 1px solid rgba(0, 0, 0, 0.214);
+  border-radius: 4px;
+}
```

Comment:
- [ ] **OPTIONAL**
Kindly combine these two selectors because they're sharing almost the same rules, the DRY rule is about don't repeat yourself, and this is for cleaner and more optimized code 😄 🙏

_You can find about best practices right [here](https://speckyboy.com/good-bad-css-practices/)._

>Please apply these changes to all similar situations through your code files!